pkg/volume/glusterfs/glusterfs.go

```diff
@@ -819,6 +840,7 @@ func (p *glusterfsVolumeProvisioner) Provision() (*v1.PersistentVolume, error) {
 		"kubernetes.io/createdby":           heketiAnn,
 		glusterTypeAnn:                      "file",
 		"Description":                       glusterDescAnn,
+		v1.MountOptionAnnotation:            "auto_unmount",
```

Comment:
Hmm, so what happens if we add support for specifying mount-options in storageclasses and then this value gets overridden? 

I think, we should have added this at mount time rather than as a annotation here.